src/test/java/uk/gov/hmcts/reform/unspec/helpers/DateFormatHelperTest.java

```diff
@@ -13,19 +14,25 @@
 
 class DateFormatHelperTest {
 
-    @Test
-    void shouldFormatDateTimeToExpectedFormat() {
-        LocalDateTime now = LocalDateTime.of(2999, 1, 1, 9, 0, 0);
+    @Nested
+    class LocalDateTimeFormat {
+        @Test
+        void shouldReturnExpectedDateTimeFormat_whenValidFormatIsPassed() {
+            LocalDateTime now = LocalDateTime.of(2999, 1, 1, 9, 0, 0);
 
-        assertThat(formatLocalDateTime(now, DATE_TIME_AT))
-            .isEqualTo("9:00am on 1 January 2999");
+            assertThat(formatLocalDateTime(now, DATE_TIME_AT))
+                .isEqualTo("9:00am on 1 January 2999");
+        }
     }
 
-    @Test
-    void shouldFormatDateToExpectedFormat() {
-        LocalDate now = LocalDate.of(2999, 1, 1);
+    @Nested
+    class LocalDateFormat {
+        @Test
```

Comment:
think no space here should fail checkstyle now?

Reply:
yes should have. will look into that separately.